lib/pageflow/user_mixin.rb

```diff
@@ -9,7 +9,7 @@ module UserMixin
     include Suspendable
 
     included do
-      belongs_to :account, :class_name => 'Pageflow::Account'
+      belongs_to :account, :counter_cache => true, :class_name => 'Pageflow::Account'
```

Comment:
Yes. New or changed code should use new hash syntax. That way we can migrate the code base step by step.